### PR TITLE
[risk=no][RW-9315] Get rid of Clarity Styling and Copy Over Relevant Bits

### PR DESCRIPTION
**Pull request by evrii**
Description:
In this initial PR, I am removing the Clarity stylesheet, but I am trying to preserve existing functionality as much as possible. I did this by copying over sections of relevant styling from the Clarity stylesheet to our main stylesheet.

In order to keep the scope of this PR small, I minimized the amount of optimizations that I did when combining the two stylesheets. This means that our stylesheet will be ripe for optimization in the near future.
Call out Clarity in CSS.

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have added explanatory comments where the logic is not obvious
- [ ] I have run and tested this change locally, and my testing process is described here
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later
- [ ] If this includes an API change, I have run the E2E tests on this change against my local server with [yarn test-local](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples) because this PR won't be covered by the CircleCI tests 
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P)
- [ ] If this change impacts deployment safety (e.g. removing/altering APIs which are in use) I have documented these in the description
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and updated the appropriate API consumers
  * AoU UI
  * [Perf tests](https://github.com/broadinstitute/mcnulty/blob/develop/src/test/scala/services/AoU.scala)
  * [Researcher Directory export](https://github.com/all-of-us/workbench/wiki/Researcher-Directory-(RDR-export))
  * Cron tasks - for Offline*Controllers
  * SumoLogic - for EgressAlert

### CODE REVIEW

**Comment by Peter-Lavigne** [comment]
Link colors are sometimes different. You can see this at `http://localhost:4200/demographic-survey`, specifically the link "defined by NIH Diversity in Extramural Programs?"

Relevant styling in prod:
image

**Comment by evrii** [comment]
> Link colors are sometimes different. You can see this at `http://localhost:4200/demographic-survey`, specifically the link "defined by NIH Diversity in Extramural Programs?"
> 
> Relevant styling in prod: image

Good catch. Thanks Peter.

File: ui/src/styles.css [hunk 1]
```diff
@@ -1,3 +1,135 @@
...
+}
+
+section {
+  display: block;
+}
+
+h1,
+h2,
+h3,
+h4 {
```
Comment (dmohs) [inline ui/src/styles.css:75]:
```suggestion
h1, h2, h3, h4 {
```

**Comment by dmohs** [comment]
Check: https://pr-6-dot-all-of-us-workbench-test.appspot.com/admin/users

**Review by Peter-Lavigne — APPROVED** [review]
💯

**Comment by Peter-Lavigne** [comment]
I compared this branch with prod on several pages side-by-side. Everything I've tried looks and functions the same.

File: ui/src/styles.css (hunk 1 above)
Comment (evrii) [reply ui/src/styles.css:75]: Codacity was giving me an error when I did it this way

**Comment by evrii** [comment]
> I compared this branch with prod on several pages side-by-side. Everything I've tried looks and functions the same.

Thank you for your help going through that.

**Comment by evrii** [comment]
> Check: https://pr-6-dot-all-of-us-workbench-test.appspot.com/admin/users

Thanks Mohs, I added a missing style that appears to fix the table problem. Thanks for finding that.

**Comment by evrii** [comment]
> Looks good. Ran through quite a bit of the app in https://pr-6-dot-all-of-us-workbench-test.appspot.com/.

Thank you for running through that.

**Pull request merged**